src/query/index.ts

```diff
@@ -98,14 +105,17 @@ const filterToApiQueryFilter = (_filters: QueryStoreFilter[], timezone = 'UTC')
 export class QueryHelper {
     private static timezone: ComputedRef<string> | undefined;
 
+    private static referenceStore: ComputedRef<Object> | undefined;
+
     private _keyMap: Record<string, KeyItem> = {};
 
     private _filters: QueryStoreFilter[] = [];
 
     private _orFilters: QueryStoreFilter[] = [];
 
-    static init(timezone: ComputedRef<string>) {
+    static init(timezone: ComputedRef<string>, referenceStore: ComputedRef<Object>) {
         QueryHelper.timezone = timezone;
+        QueryHelper.referenceStore = referenceStore ?? undefined;
```

Comment:
QueryHelper.referenceStore가 비어있으면 항상 nullable 체크를 해야하니, 빈 객체로 initiate 하는건 어떨까요?
```suggestion
    static init(timezone: ComputedRef<string>, referenceStore: ComputedRef<Object> = {}) {
        QueryHelper.timezone = timezone;
        QueryHelper.referenceStore = referenceStore;
```